src/main/java/ch/njol/skript/effects/EffDetonate.java

```diff
@@ -0,0 +1,56 @@
+package ch.njol.skript.effects;
+
+import org.bukkit.entity.*;
+import org.bukkit.entity.minecart.ExplosiveMinecart;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+
+@Name("Detonate Entity")
+@Description("Automatically detonates an entity.\n Accepted entities are 'Fireworks, TNT Minecarts and WindCharges")
```

Comment:
```suggestion
@Description("Immediately detonates an entity. Accepted entities are fireworks, TNT minecarts and wind charges.")
```
Obviously add creepers or whatever to the description if they get added to this PR.